src/jackdaw/test/transports/mock.clj

```diff
@@ -56,7 +56,12 @@
     {:topic (.topic r)
      :key (.key r)
      :value (.value r)
-     :partition (or (.partition r) -1)}))
+     :partition (or (.partition r) -1)
+     :matt-was-here true
```

Comment:
Presume this was left-over from testing? :-)

LGTM otherwise.

Reply:
Doh! yes :-)
Hope you are doing well Andy!